src/actions/index.js

```diff
@@ -67,7 +67,7 @@ export const openFolder = (folderId, context = FILES_CONTEXT) => {
       context,
       folder: Object.assign(extractFileAttributes(folder), {
         parent: extractFileAttributes(parent)}),
-      files: folder.relations('contents').map(
+      files: (folder.relations('contents') || []).map(
```

Comment:
I think a Rest API must be consistent in the type of data it sends. If it should be an array, the array should be empty if there is no data, and a null value must declare something else like an error or an inconsistent data.